presto-sqlserver/src/test/java/io/prestosql/plugin/sqlserver/SqlServerQueryRunner.java

```diff
@@ -0,0 +1,107 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.plugin.sqlserver;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import com.google.common.collect.Streams;
+import io.airlift.tpch.TpchTable;
+import io.prestosql.Session;
+import io.prestosql.metadata.QualifiedObjectName;
+import io.prestosql.plugin.tpch.TpchPlugin;
+import io.prestosql.spi.security.Identity;
+import io.prestosql.testing.QueryRunner;
+import io.prestosql.tests.DistributedQueryRunner;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.Set;
+
+import static com.google.common.collect.ImmutableSet.toImmutableSet;
+import static io.airlift.testing.Closeables.closeAllSuppress;
+import static io.prestosql.plugin.tpch.TpchMetadata.TINY_SCHEMA_NAME;
+import static io.prestosql.testing.TestingSession.testSessionBuilder;
+import static io.prestosql.tests.QueryAssertions.copyTable;
+import static java.util.Locale.ENGLISH;
+
+public final class SqlServerQueryRunner
+{
+    public static final String CATALOG = "sqlserver";
+    private static final String TEST_SCHEMA = "dbo";
+
+    private SqlServerQueryRunner()
```

Comment:
- `.... () {}`
- move above everything else in the class, this isn't a real constructor